content/guides/outliers.md

```diff
@@ -80,75 +80,51 @@ There are two different outlier detection algorithms you can use on your data: D
 
-The [Median Absolute Deviation](https://en.wikipedia.org/wiki/Median_absolute_deviation) (MAD) algorithm is a robust method of calculating variability among data. It is designed to be usable for both normal and non-normal distributions.
+<img src="/static/images/outliers/outliers-mad-az.png" style="width:100%; border:1px solid #777777"/>
 
-We use MAD to calculate an *outlier_factor* which we will compare to points from a series. If the percentage of points in the series that are above the *outlier_factor* is greater than some given threshold, then that series will be considered an outlier.
+The scale parameter should be tuned depending on expected variability of the data. For example if data is generally within a small range of values then this should be small. On the other hand if points can vary greatly then you want a higher scale so these variabilities do not trigger a false positive.
 
-The calculation of the outlier_factor is as such:
+### DBSCAN vs. MAD
 
-~~~ python
-    # Calculate the MAD across all series.
-    series_values = flatten_series(series_list)
-    median = median(all_values)
-    deviations = abs(x - median for x in series_values)
-    mad = median(deviations)
+So which algorithm should you use? For most outliers, both algorithms will perform well at the default settings. However, there are subtle cases where one algorithm is more appropriate than the other.
 
-    # Outlier factor is calculated from the scale and standard error.
-    # We divide by a normalization constant of 0.6745 to make the scale
-    # 1.0, 2.0, 3.0 represent standard deviations for metrics drawn from a normal
-    # distribution.
-    outlier_factor = (scale * mad) / 0.6745
-~~~
+In the following image, we see a group of hosts flushing their buffers together while one host is flushing its buffer slightly later. DBSCAN picks this up as an outlier whereas MAD does not. This is a case where we would prefer to use MAD, as we don’t care about when the buffers get flushed. The synchronicity of the group is just an artifact of the hosts being restarted at the same time. On the other hand, if instead of flushed buffers, the metrics below represented a scheduled job that actually should be synchronized across hosts, DBSCAN would be the right choice.
 
-#### Parameters
+<img src="/static/images/outliers/outliers-flushing.png" style="width:100%; border:1px solid #777777"/>
 
-MAD accepts the following parameters:
+### Setting up alerts
+When setting up an outlier alert, an important parameter is the size of the time window. If the window size is too large, by the time an outlier is detected, the bad behavior might have been going on for longer than one would like. If the window size is too short, the alerts will not be as resilient to spikes.
```

Comment:
> ...the alerts will not be resilient to spikes.

Would "normal volatility" be better than "spikes." One might ask what the usefulness of outlier detection is if it ignores spikes.